src/GraphQL.php

```diff
@@ -55,33 +55,13 @@ public function schema($schema = null): Schema
 
         $this->clearTypeInstances();
 
-        foreach ($this->getTypes() as $name => $type) {
-            $this->type($name);
-        }
-
```

Comment:
I forgot to mention that yesterday I also removed this code because I also had the feeling it wasn't doing much 😄

Reply:
Tested on a live system and failed.

The type loader receives the name from the Types name-attribute wich can be different from the alias used in config. That mean we have to check every file for the name.

One way, could be to make a artisan graphql cache command that mapped the type names to the classes so it was easily found.

But I might be missing something...

Reply:
> One way, could be to make a artisan graphql cache command that mapped the type names to the classes so it was easily found.

This direction seem appropriate to explore!

Basically we need to figure out what and how can we cache it and then make a config entry for this.

With such a thing in place, if you deploy code (changes to GraphQL type system in some way) and don't clear the cache it would probably break. Which is _fine_ because the recommendation _is_ to clear the cache after deployment.